src/types/rooms.ts

```diff
@@ -1,20 +1,24 @@
-export interface BaseRoom {
+export enum RoomTypes {
+  unclickable = "UNCLICKABLE",
+}
```

Comment:
This is what I was wanting 👍🏻 :)

May consider it OOS, but https://github.com/sparkletown/internal-sparkle-issues/issues/505 will also basically want to add a type here too (required for the Monday event)

Reply:
> May consider it OOS, but sparkletown/internal-sparkle-issues#505 will also basically want to add a type here too (required for the Monday event)

That comment was wrong, as it was a new venue template we needed there, not a new room type.